components/onboarding/WelcomeScreen.vue

```diff
@@ -0,0 +1,152 @@
+                        rounded-full animate-bounce z-10"
+                        :style="{ animationDuration: '1.5s' }"
+                        @click="expandBackground"
+                    >
+                        <!-- Button text -->
+                        <span class="text-xl font-bold text-primary-text-inverted">
+                            {{ t('onboarding.letsGoButtonText') }}
+                        </span>
+                        <!-- Right-pointing Arrow SVG -->
+                        <svg
+                            class="w-6 h-6 pt-1 text-primary-text-inverted transform
+                                group-hover:translate-x-1 transition-transform"
+                            fill="none"
+                            stroke="currentColor"
+                            viewBox="0 0 24 24"
+                            xmlns="http://www.w3.org/2000/svg"
+                        >
+                            <path
+                                stroke-linecap="round"
+                                stroke-linejoin="round"
+                                stroke-width="3"
+                                d="M9 5l7 7-7 7"
+                            />
+                        </svg>
+                    </button>
+                </div>
+                <!-- Background wave svg -->
+                <div class="w-full absolute bottom-0">
+                    <svg
+                        viewBox="0 0 375 56"
+                        class="w-full z-10 transition-all duration-500ease-out"
+                        :class="{ 'h-0': isBackgroundExpanding }"
+                        width="100%"
+                        height="56"
+                        preserveAspectRatio="none"
+                        :style="{ display: 'block' }"
+                        @mouseenter="waveHover = true"
+                        @mouseleave="waveHover = false"
+                    >
+                        <!-- Subtly morph the wave path when hovered -->
+                        <path
+                            :d="waveHover
+                                ? 'M375 0V56H0V25C50 33 125 70 187.5 50C250 30 320 -10 375 25Z'
+                                : 'M375 0V56H0V18.5C46.5 33 120 59 187.5 45C255 31 328.5 -13.5 375 18.5Z'"
+                            class="fill-primary-bg"
+                            :style="{ transition: 'd 0.4s cubic-bezier(.6,0,.4,1)' }"
+                        />
+                    </svg>
+                </div>
+            </div>
+            <!-- Bottom half of screen -->
+            <div
+                class="h-full flex items-center justify-center transition-opacity duration-500"
+                :class="{ 'opacity-0': isBackgroundExpanding }"
+            >
+                <!-- Faded background findadoc character team -->
+                <SVGCharactersTogetherWelcomeScreen
+                    role="img"
+                    alt="Findadoc character team"
```

Comment:
Nit: We should replace all `findadoc` instances with `Find a Doc, Japan` since that is our official name.

Reply:
honestly, it's annoying to add the comma and japan part each time, but I'll try to add it 😆 the alt tag is user facing, so it's fair. 
I'm assuming you don't think we need it on the code parts though like findadoccommajp/web, right?